baseMigrations/migrate.js

```diff
@@ -897,7 +897,7 @@ function _createSSHKeysSystemIntegration(bag, next) {
 
   var systemIntegration = {
     name: 'sshKeys',
-    masterName: 'ssh-key',
+    masterName: 'sshKey',
```

Comment:
How will this skip if the `sshKeys` system integration was already created using the `ssh-key` master integration?